Packs/CaseManagement-Generic/ReleaseNotes/1_4_2.json

```diff
@@ -0,0 +1 @@
+{"breakingChanges":true,"breakingChangesNotes":"The scripts AssignToMeButton, GenerateSummaryReportButton, LinkIncidentsButton were moved to the *CommonScripts* pack. Make sure to update the *CommonScripts* pack to the latest version in order to use these scripts."}
```

Comment:
is this change (under `Packs`, rather than the testdata) intended, or were they just for reproducing the bug?

Reply:
Intended for testing purposes.

Reply:
please ping me when it's removed & PR is ready

Reply:
I meant - It's used in my unit-test. It's needed, and it's going to stay. :-)

Once you commented I can't remove you. Sorry. :-)